### ISSUE TITLE
Multi user switching for quick permisions testing

### ISSUE THREAD

**Issue by JasCodes** [parse-community/parse-dashboard#2056]
### New Feature / Enhancement Checklist

- [x] I am not disclosing a [vulnerability](https://github.com/parse-community/parse-server/blob/master/SECURITY.md).
- [x] I am not just asking a [question](https://github.com/parse-community/.github/blob/master/SUPPORT.md).
- [x] I have searched through [existing issues](https://github.com/parse-community/parse-dashboard/issues?q=is%3Aissue).

### Current Limitation
Even if I try to open another tab, can't have more then one user session.
 
### Feature / Enhancement Description
- Allows to login multiple users.
- Allow to logout all users with single click.
- Easy switching between logged in users.

### Example Use Case
Testing permissions

**Comment by JasCodes** [comment]
@mtrezza feedback please

**Comment by davimacedo** [comment]
Have you tried the Browser As User option?

**Comment by JasCodes** [comment]
As, but it doesn't remember more than one user. And its very cumbersome to switch between multiple users.

**Comment by davimacedo** [comment]
I understand your issue, but I am not sure how the UI would work for what you want. Any suggestion? Maybe can you send a mockup?

**Comment by JasCodes** [comment]
<img width="308" alt="pasre dashboard user browsing mockup" src="https://user-images.githubusercontent.com/1231593/156542973-b8dc5e87-1b33-4ea1-92f4-9fc8ffd1a813.png">

We can probably design a bit better. Also when you refresh the dashboard it auto resets use the master key all the states should be peristed.

@davimacedo Also I am able to edit CLP from the regular users, and thinking from behind the scene dashboard is using master key. Am I right or as I am new to parse, I am getting security wrong?

**Comment by davimacedo** [comment]
Yes. You are right. Parse Dashboard uses the master key. The idea of browse as other multiple users makes sense to me. Would you be willed to send a PR?

**Comment by mtrezza** [comment]
I think that would certainly be a great further development of the Browse-as-user feature. Excited to see your PR!

**Comment by JasCodes** [comment]
@mtrezza For sure, but I am really new to parse and unfamiliar to the codebase. Though I will love to be a regular code contributor as started using it with two commercial projects. And its always good to give back. Can somebody from the community who knows the codebase do a quick call with me? Is that possible? Are there any web meetups etc?

**Comment by mtrezza** [comment]
What you could do is take a look at https://github.com/parse-community/parse-dashboard/pull/1750 that added the feature. The code and the discussion thread should give you an idea of how this feature came to be. Then you could make a suggestion on how you'd extend that and get community feedback.

**Comment by ssamuli** [comment]
Looking at this issue and PR #1750, I don't understand why none of these "Browse as user" related PRs have added the possibility to login to whole parse dashboard with a _User (instead of the current hard coded users array defined in dashboard config) - would that be an easy feature to add or is there a good reason to not allow it?

**Comment by mtrezza** [comment]
@ssamuli Good question, the reason is likely the effort to add such a feature. These are two separate authentication systems. See https://github.com/parse-community/parse-dashboard/issues/2311 if you want to discuss that topic.
